docker-compose.test.yml

```diff
@@ -33,10 +33,14 @@ services:
     volumes:
       - ./app:/app
     depends_on:
-      - db
-      - rabbitmq
-      - redis
+      db:
```

Comment:
Les tests échouaient parce que le service web était prêt avant que la DB soit véritablement up. J'ai trouvé la solution dans cet article de [blog](https://laurent-bel.medium.com/waiting-for-postgresql-to-start-in-docker-compose-c72271b3c74a).